packages/mini-browser/src/browser/mini-browser-open-handler.ts

```diff
@@ -273,7 +273,7 @@ export class MiniBrowserOpenHandler extends NavigatableWidgetOpenHandler<MiniBro
 
     protected getSourceUri(ref?: Widget): URI | undefined {
         const uri = ref instanceof MiniBrowser && ref.getResourceUri() || undefined;
-        if (!uri || uri.scheme === 'http' || uri.scheme === 'https') {
```

Comment:
Do we really want to remove the checks for the HTTP(S) scheme though? Should we rather add the new check on top of the old ones?

Reply:
IIRC the mini-browser can still display HTTP(S) content and now it will show the "open source" button, no?

Reply:
You're right, we should probably add this new check in addition to the old ones.

Reply:
If you follow the chain of uri, you'll see that it comes from `widgetOptions`. This URI is only internal to Theia, and doesn't contain any http, or https scheme. Probably what you're referring to is `startPage`. Mini browser can display http(s) content, but only when it's opened in preview mode. Also, the http(s) content's link will be inside `startPage` MiniBrowserProps, and not URI. Hope that clears things up!